student/src/main/java/tds/student/web/handlers/MasterShellHandler.java

```diff
@@ -136,6 +137,9 @@
   private ITDSLogger             _tdsLogger;
 
   @Autowired
+  private StudentEventLogger     _eventLogger;
```

Comment:
If we autowire a request-scoped property, does that mean this class needs to also be request-scoped?